SPECS/maven/maven.spec

```diff
@@ -1,65 +1,109 @@
+# Switch "sources_generation" to 1 when running a package build to generate cached sources for regular builds.
+%define sources_generation 0
+%define m2_cache_tarball_name apache-%{name}-%{version}-m2.tar.gz
+%define licenses_tarball_name apache-%{name}-%{version}-licenses.tar.gz
+
+%if ! 0%{?sources_generation}
+%define offline_build -o
```

Comment:
Seems like we ALWAYS want that set.  doesn't hurt to have the option here, but you could reduce the complexity.

Reply:
We don't want to have this set when `sources_generation` is equal to 1. This option makes Maven build in offline mode and we need it to use the network when we generate the tarballs.